fake.go

```diff
@@ -519,6 +521,177 @@ func (fake *Fake) Dump() string {
 	return buf.String()
 }
 
+// extractComment removes comment section from a given string and returns it together with comment string.
+// If a comment wasn't found, an empty comment string will be returned.
+func extractComment(s string) (string, string) {
+	if !strings.Contains(s, "comment") {
+		return s, ""
+	}
+	// find comment section with colon
+	reg := regexp.MustCompile(`comment.*".*" ?;?`)
+	// extract comment section and leave just the string
+	comment := reg.FindStringSubmatch(s)[0]
+	commentReg := regexp.MustCompile(`"[^"]+"`)
+	comment = commentReg.FindStringSubmatch(comment)[0]
+	comment = strings.Trim(comment, "\"")
+	// remove comment section from the initial string
+	res := reg.ReplaceAllString(s, "${1}")
+	return strings.TrimSpace(res), comment
+}
+
+// ParseNFTablesDump can parse a dump for a given nft instance.
+// It expects one table declaration and the same nftables family in all rules based on a given nft.nftContext.
+// It lacks some error handling, and the best way to verify that everything important was properly parsed is to
+// compare given data with nft.Dump() output.
```

Comment:
Eventually, it ought to be able to parse anything that `fake.Dump` might output. It's less important to be able to parse arbitrary random nft commands...